src/core_ocean/Registry.xml

```diff
@@ -1302,6 +1302,12 @@
 					possible_values="'linear_x', 'linear_y', 'linear_arb_rot', 'power_x', 'power_y', 'power_arb_rot', 'sin_arb_rot', 'sph_solid_body', 'sph_Williamson', 'sph_uCosCos', 'sph_vCosCos', 'sph_ELonLon_CosCos', 'sph_ELatLat_CosCos', 'sph_ELonLat_CosCos'"
 		/>
 	</nml_record>
+	<nml_record name="init_mode_vert_levels" mode="init">
+		<nml_option name="config_vert_levels" type="integer" default_value="-1" units="unitless"
```

Comment:
I had to leave this flag in the main Registry file because it is used to fill in a dimension.

Reply:
I thought about this and looked through the code.  I can't come up with a cleaner solution than this either.